scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,100 @@
+# IPC gullwing generator
```

Comment:
Thanks for taking this on!!!

- I don't see anywhere in here where it is specified that the datasheet dimensioning is to be used.
- I wonder if an image of an applicable package would be useful?
- Since many of the sections will apply to many of the generators, is there a way to have "master" sections of this doc which can be stitched together, with package-specific sections, to make a complete readme? Having the same content in a dozen files seems like a potential maintenance nightmare.

Reply:
Well i reduced it down to two generators so there are not really that many readmes that can share data in any useful way. (And the no lead style packages are way too different to merit the extra work for the little gain.)

Reply:
I looked into it: github readme file format does not support includes.

Reply:
Did I miss anywhere that states to get the dimensional information from a datasheet or package drawing? You recall a recent PR where the contributor tried to use the footprint dimensions.

Reply:
added